src/diffusers/loaders/autoencoder.py

```diff
@@ -103,6 +104,13 @@ def from_single_file(cls, pretrained_model_link_or_path, **kwargs):
         use_safetensors = kwargs.pop("use_safetensors", True)
 
         class_name = cls.__name__
+
+        if (config_file is not None) and (original_config_file is not None):
+            raise ValueError(
+                "You cannot pass both `config_file` and `original_config_file` to `from_single_file`. Please use only one of these arguments."
+            )
+
+        original_config_file = original_config_file or config_file
```

Comment:
Do we need to do a type-check on `original_config_file` and `config_file`?

Reply:
What other type could be passed in here? Dict? I don't believe we supported passing in the loaded config dict before.

Reply:
Yeah makes sense. Subsequent errors should be caught during loading the file. So, all good.